docker-api/src/main/java/com/yahoo/vespa/hosted/dockerapi/ContainerStats.java

```diff
@@ -104,13 +106,23 @@ public CpuStats(CpuStatsConfig cpuStats) {
             this.totalUsage = cpuStats.getCpuUsage().getTotalUsage();
             this.usageInKernelMode = cpuStats.getCpuUsage().getUsageInKernelmode();
             this.throttledTime = cpuStats.getThrottlingData().getThrottledTime();
+            this.throttlingActivePeriods = cpuStats.getThrottlingData().getPeriods();
+            this.throttledPeriods = cpuStats.getThrottlingData().getThrottledPeriods();
         }
 
         public int getOnlineCpus() { return this.onlineCpus; }
         public long getSystemCpuUsage() { return this.systemCpuUsage; }
         public long getTotalUsage() { return totalUsage; }
         public long getUsageInKernelMode() { return usageInKernelMode; }
+
+        /** Total CPU time processes in this container were throttled for */
```

Comment:
This is in nanoseconds, right? Document or make clear in method name?

Reply:
Added